File: Backup/PlaceYourBets/SubmittedForm.vb
Public Class PredictionsSubmitted

    Private userList As New List(Of String)
    Private submittedList As New List(Of String)

    Private Sub ShutdownButton_Click(sender As System.Object, e As System.EventArgs)
        Application.Exit()
    End Sub

    Private Sub MainMenuButton_Click(sender As System.Object, e As System.EventArgs) Handles MainMenuButton.Click
        MenuForm.Show()
        Me.Close()
    End Sub

    Private Sub ReviewForm_Load(sender As System.Object, e As System.EventArgs) Handles MyBase.Load

        Dim fb As New FixtureBuilder
        userList = fb.getUsers()
        submittedList = fb.usersSubmitted()

        Me.UserPredictionsFlowLayoutPanel.Enabled = True
        Me.UserPredictionsFlowLayoutPanel.Visible = True
        Me.NavigationGroupBox.Visible = True
        Me.MainMenuButton.Visible = True

        For Each user As String In userList
            Dim review As New UserPredictionsUserControl(user)
            Me.UserPredictionsFlowLayoutPanel.Controls.Add(review)

            If submittedList.Contains(user) Then
                review.submittedLabel.BackColor = Color.LightGreen
                review.submittedLabel.Text = "Submitted"
            End If

        Next

    End Sub

    Private Sub UserPredictionsFlowLayoutPanel_Paint(sender As System.Object, e As System.Windows.Forms.PaintEventArgs) Handles UserPredictionsFlowLayoutPanel.Paint

    End Sub
End Class

File: PlaceYourBets.ConvertedToC#/SubmittedForm.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Drawing;
using System.Diagnostics;
using System.Windows.Forms;
using System.Linq;
using System.Xml.Linq;
namespace PlaceYourBets
{
	public partial class PredictionsSubmitted
	{

		private List<string> userList = new List<string>();

		private List<string> submittedList = new List<string>();
		private void ShutdownButton_Click(System.Object sender, System.EventArgs e)
		{
			Application.Exit();
		}

		private void MainMenuButton_Click(System.Object sender, System.EventArgs e)
		{
			My.MyProject.Forms.MenuForm.Show();
			this.Close();
		}


		private void ReviewForm_Load(System.Object sender, System.EventArgs e)
		{
			FixtureBuilder fb = new FixtureBuilder();
			userList = fb.getUsers();
			submittedList = fb.usersSubmitted();

			this.UserPredictionsFlowLayoutPanel.Enabled = true;
			this.UserPredictionsFlowLayoutPanel.Visible = true;
			this.NavigationGroupBox.Visible = true;
			this.MainMenuButton.Visible = true;

			foreach (string user in userList) {
				UserPredictionsUserControl review = new UserPredictionsUserControl(user);
				this.UserPredictionsFlowLayoutPanel.Controls.Add(review);

				if (submittedList.Contains(user)) {
					review.submittedLabel.BackColor = Color.LightGreen;
					review.submittedLabel.Text = "Submitted";
				}

			}

		}


		private void UserPredictionsFlowLayoutPanel_Paint(System.Object sender, System.Windows.Forms.PaintEventArgs e)
		{
		}
		public PredictionsSubmitted()
		{
			Load += ReviewForm_Load;
			InitializeComponent();
		}
	}
}
